jobs/config.json

```diff
@@ -892,6 +892,16 @@
   ]
 },
 
+"ci-kubernetes-e2e-prow-canary": {
+  "scenario": "kubernetes_prow",
```

Comment:
Prow currently adds the service account json file, but doesn't have any SSH keys.